src/mono/browser/runtime/logging.ts

```diff
@@ -35,6 +35,7 @@ export function mono_log_error(msg: string, ...data: any) {
 }
 
 export const wasm_func_map = new Map<number, string>();
+const wasm_pending_symbol_tables = new Array<string>();
```

Comment:
There is only one such symbol file, we don't need a list here.

Reply:
Should we fail fast if there's a second one, then?

Reply:
yep